fastlane/docs/Guide.md

```diff
@@ -125,16 +118,22 @@ error do |lane, exception|
 end
 ```
 
-You can already try running it, put a line (`say "It works"`) to the `:inhouse` lane and run
+You can already try running `fastlane`, add this to your `Fastfile`
```

Comment:
"You can quickly try running `fastlane` by adding the following to your `Fastfile`: